hicommonwealth/commonwealth
# Added boilerplate for "my community stakes" page

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/7050

## Description of Changes
- Added boilerplate code for "my community stakes" page
- Added `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` to hide the "my community stakes" page components

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` flag in `.env`
- Visit `/myCommunityStake`
- Verify you see this UI, and verify this is responsive across different screen sizes
<img width="1386" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/bb248944-9012-492c-9c70-1e1534c3db92">


## Deployment Plan
N/A

## Other Considerations
N/A
